portal-impl/src/com/liferay/portlet/documentlibrary/service/impl/DLFileEntryLocalServiceImpl.java

```diff
@@ -2771,6 +2790,89 @@ private long _getValidFileEntryTypeId(
 		}
 	}
 
+	private void _inheritRolesPermissions(
+			long parentFolderId, DLFileEntry dlFileEntry)
```

Comment:
no need to parentFolderId in here because it can get from dlFileEntry

Reply:
That is the folder which the file entry inherit the permissions. I will change the name. Thanks